src/domain/shared/components/SimpleCard.tsx

```diff
@@ -7,7 +7,7 @@ import IconLabel from './IconLabel';
 import { LinkWithState } from '../types/LinkWithState';
 
 const ElevatedPaper = withElevationOnHover(Paper);
-export const SIMPLE_CARD_THEME_WIDTH = 25;
+export const SIMPLE_CARD_THEME_WIDTH = 32;
```

Comment:
I think it's better to add this constant to the theme instead so everything can use it.

Reply:
good point, didn't know about these constants. thanks